Code/skyrim_ui/src/app/services/player-list.service.ts

```diff
@@ -120,6 +123,18 @@ export class PlayerListService {
     });
   }
 
+  private onPlayerPartyLeave() {
+    this.playerPartyLeaveSubscription = this.clientService.partyLeftChange.subscribe((leavingPlayerId: number) => {
+      const playerList = this.getPlayerList();
+
+      if (playerList) {
+        this.getPlayerById(leavingPlayerId).hasInvitedLocalPlayer = false;
```

Comment:
need to null check here, since a player could've just left the game, not necessarily a party